fix: LEAP-240: Prevent creating region on clicking outside selected one

robot-ci-heartex opened the pull request:
Hi @Gondragos!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7565497444) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1667

Gondragos committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7565497444

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7565497444

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7565522678

Gondragos committed:
Merge branch 'develop' into 'fb-leap-240/fix-click-outside'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7629026402

Gondragos committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7629026402

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7629074776

Gondragos committed:
Merge branch 'develop' into 'fb-leap-240/fix-click-outside'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7698726433

Gondragos committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7698726433

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7698751218

Gondragos commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7699434023)
> Successfully merged: Already up to date.

Commit:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7699619466

Gondragos commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7701405045)
> Successfully merged:  create mode 100644 docs/scripts/replaceContent.js

Gondragos committed:
Merge branch 'develop' into 'fb-leap-240/fix-click-outside'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7701405045

Gondragos commented:
/frontend build lsf
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7701893881)
> Successfully pushed new changes
>  heartexlabs/label-studio-frontend@923c09cbbd8c03468e0c45445086f0e817d99f0c

Gondragos committed:
[frontend] Get build lsf

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7701893881
